src/api/Intelligence.js

```diff
@@ -75,6 +75,7 @@ class Intelligence extends Base {
             suggestionsResponse = await this.xhr.post({
                 url,
                 data: request,
+                id: `file_${request.items[0].id}`,
```

Comment:
i see we do this in the `ask` method as well but how important is this `id` field? if `items` has multiple files would this cause conflicts or unexpected behavior?

Reply:
`id` is used internally by post to get auth token, it can by an array, but for this endpoint we only use one file. That is whybI take first element.